src/Cms/Form/Element/MultiField.php

```diff
@@ -170,6 +170,11 @@ private function renderListElement(?array $itemValues = null, string $index = '*
             $element->setName($this->getBaseName() . '[' . $index . '][' . $element->getBaseName() . ']');
             $element->setValue($itemValues[$element->getBaseName()] ?? null);
             $element->setErrors($this->_elementErrors[$index][$element->getBaseName()] ?? []);
+
+            if ($element instanceof Checkbox) {
+                $element->getValue() ? $element->setChecked() : $element->setChecked(false);
```

Comment:
ja bym to zapisal tak $element->setChecked(!!$element->getValue()) ale tak jak jest tez moze byc

Reply:
efekt byłby ten sam co w:
$element->setChecked((bool) $element->getValue())
a jakoś średnio przepadam za castowaniem w miejscach, w których dane pochodzą od nas z kodu i powinny być porządnie typowane już wcześniej - tu nie powinno trafić żadne 1, "1", "on" itp.
w cms pole checkbox ma w domyśle wartość `true`, więc de facto getValue z checkboxa zwraca `?bool` 
a jeśli by nie zwracało, to trzeba poprawić samo pole :)

Reply:
podwójne zaprzeczenie? hmmm